airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/cursor/default_file_based_cursor.py

```diff
@@ -48,11 +48,21 @@ def add_file(self, file: RemoteFile) -> None:
                 )
 
     def get_state(self) -> StreamState:
-        state = {
-            "history": self._file_to_datetime_history,
-        }
+        state = {"history": self._file_to_datetime_history, "_ab_source_file_last_modified": self._get_cursor()}
```

Comment:
Hmm that isn't actually covered by the CATs. Do you know what we usually do in that situation?

Reply:
Actually I think I'll leave it as-is for now, since we don't actually use the cursor field for deciding what to sync; it's just being used as a tool for integration tests.